README.md

```diff
@@ -144,76 +141,72 @@ Keep in mind that companies may have job postings on their own site that are not
 * [Full Stack Developer](http://www.referralsaasquatch.com/careers/)
 
-#### [RevenueWire](https://www.revenuewire.com/)
+#### [RevenueWire](https://www.revenuewire.com/) (👩‍💻 Co-Op / Intern Friendly)
 * [Sales Specialist](https://www.revenuewire.com/careers/#tab-2)
 * [PHP Platform Developer](http://www.revenuewire.com/careers/)
 
-#### [RingPartner](http://ringpartner.com/)
+#### [RingPartner](http://ringpartner.com/) (👩‍💻 Co-Op / Intern Friendly)
 * [Node Micro-Service Developer](https://ringpartner.bamboohr.com/jobs/view.php?id=5)
 
-#### [Rooof](http://www.rooof.com)
+#### [Rooof](http://www.rooof.com) (👩‍💻 Co-Op / Intern Friendly)
 * [Quality Assurance Engineer (Software)](http://www.rooof.com/qa-engineer)
 
 #### [Ruboss](http://ruboss.com/)
 
-#### [SeafoodX](https://www.seafoodx.io/)
+#### [SeafoodX](https://www.seafoodx.io/) (👩‍💻 Co-Op / Intern Friendly)
 
-#### [Sendwithus](https://www.sendwithus.com)
+#### [Sendwithus](https://www.sendwithus.com) (👩‍💻 Co-Op / Intern Friendly)
 * [Full Stack Product Engineer](https://www.sendwithus.com/careers/4d98220d-516e-44a3-abcb-16290421409c)
 * [Senior Platform Engineer](https://www.sendwithus.com/careers/a6841dfb-08ff-44bf-afb2-d7a07bdcee8c)
 * [Customer Success Associate](https://www.sendwithus.com/careers/a7bb4a91-71ec-497b-8e90-95b941a74a46)
-* [Operations Coordinator](https://www.sendwithus.com/careers/f0cd5c4d-e3ca-498c-8627-997c545cd9b1)
 * [Content Marketer](https://www.sendwithus.com/careers/12d30ac4-38ee-41f7-9747-67e5a51aa694)
-* [QA Engineer](https://www.sendwithus.com/careers/0456beb4-9d30-47ca-af93-9ead23cecabb)
 * [UI/UX Designer](https://www.sendwithus.com/careers/2f92d2b4-d846-47ed-9056-88a52010421f)
 
 #### [SparkLIT](https://www.sparklit.com/)
 * [Intermediate Software Engineer, Full Stack](https://www.sparklit.com/careers/#full-stack)
 * [UI/UX Graphic Designer](https://www.sparklit.com/careers/#graphic-designer)
 
-#### [Silkstart](http://silkstart.com/about/)
+#### [Silkstart](http://silkstart.com/about/) (👩‍💻 Co-Op / Intern Friendly)
 * [Quality Assurance Developer (Test Automation)](https://www.viatec.ca/jobs/quality-assurance-developer-test-automation)
 
-#### [Stembolt](http://stembolt.com)
+#### [Starfish Medical](http://starfishmedical.com/) (👩‍💻 Co-Op / Intern Friendly)
+
+#### [Stembolt](http://stembolt.com) (👩‍💻 Co-Op / Intern Friendly)
 * [Software Developer Co-op](https://stembolt.com/careers/software-developer-co-op/)
 * [Junior Developer](https://stembolt.com/careers/junior-developer/)
 * [Intermediate/Senior Developer](https://stembolt.com/careers/intermediate-or-senior-developer/)
 * [Senior Solidus Developer](https://stembolt.com/careers/senior-solidus-developer/)
 
-#### [Stocksy United](http://www.stocksy.com/)
-* [Accounts Payable & Payroll Specialist](https://www.stocksy.com/service/hiring)
-* [Accountant](https://www.stocksy.com/service/hiring)
+#### [Stocksy United]()
+* [Accounts Payable & Payroll Specialist]()
+* [Accountant]()
```

Comment:
Once #78 is merged into `master` and this PR is rebased, reverting the Stocksy URL removal commits shouldn't be a problem.